stories/Select.stories.js

```diff
@@ -13,8 +13,19 @@ stories.addParameters({
 });
 
 stories.add('Default', () => (
-  <Select onChange={action('Select changed')}>
-    <option value="" disabled selected>
+  <Select value="" onChange={action('Select changed')}>
+    <option value="" disabled>
+      Choose your option
+    </option>
+    <option value="1">Option 1</option>
+    <option value="2">Option 2</option>
+    <option value="3">Option 3</option>
+  </Select>
+));
+
+stories.add('with Icon', () => (
+  <Select value="" icon="cloud">
```

Comment:
can we remove `value=""`? if no value is passed it still works and like this it seems this is the required interface

Reply:
I used [materialize-css docs](https://materializecss.com/select.html) for reference.
Anyway, if you wish, I can remove it.